super_rust_boy_c_binding/src/lib.rs

```diff
@@ -64,7 +64,7 @@ pub extern fn rustBoyCreate(cartridge_path: *const c_char, save_file_path: *cons
         }
     };
 
-    let instance = RustBoy::new(cart_path, save_path, UserPalette::Default, false);
+    let instance = RustBoy::new(cart_path, save_path, UserPalette::Default, true);
```

Comment:
A note for the future which you don't have to address now - but it would be cool to let the user set a palette. However there is currently no API for changing palette while the emulator is alive.

Reply:
Sounds like a cool option. Let me know when that is supported in the core API